File: 268. Missing Number/missingnumber.java
class Solution {
    public int missingNumber(int[] nums) {
        int numsSize = nums.length, expectedSum = numsSize * (numsSize + 1) / 2, actualSum = 0;

        for (int i = 0; i < numsSize; i++) {
            actualSum += nums[i];
        }

        return expectedSum - actualSum;
    }
}

File: 268. Missing Number/missingnumber.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        numsSize = len(nums)
        expectedSum = numsSize * (numsSize + 1) // 2
        actualSum = 0

        for i in range(numsSize):
            actualSum += nums[i]

        return expectedSum - actualSum
